AAI/AAIConnectProfile.md

```diff
@@ -696,22 +745,69 @@ If this information is a JWT, especially from the recommended Token Endpoint, th
 }
 ```
 
--   `iss` and `sub`: REQUIRED.
+- `iss` and `sub`: REQUIRED.
 
--   `aud`: OPTIONAL.
+- `aud`: OPTIONAL.
 
--   `iat`: REQUIRED only if JWT Otherwise OPTIONAL. Time issued.
+- `iat`: REQUIRED only if JWT Otherwise OPTIONAL. Time issued.
 
--   `exp`: REQUIRED only if JWT Otherwise OPTIONAL. Time expired.
+- `exp`: REQUIRED only if JWT Otherwise OPTIONAL. Time expired.
 
--   `<ga4gh-spec-claims>`: OPTIONAL. GA4GH Claims are generally included as
+- `<ga4gh-spec-claims>`: OPTIONAL. GA4GH Claims are generally included as
     part of one or more GA4GH standard specifications based on the scopes
     provided. Even when requested by the appropriate scopes, these GA4GH Claims
     may not be included in the response for various reasons, such as if the
     user does not have any GA4GH Claims. See
     [Authorization/Claims](#authorizationclaims) for an example of a GA4GH
     Claim.
 
+##### Passport Token Contents
+
+Passport Issuers MUST issue a Passport token conforming to the requirements in this section when a token exchange
+with the `requested_token_type=urn:ga4gh:params:oauth:token-type:passport` is successfully performed
+(as described in the [Conformance for Passport Issuers](#conformance-for-passport-issuers) section).
+
+###### Header
+
+This spec prescribes the following JWS headers for Passport tokens
+in addition to the guidelines established in [RFC7515](https://datatracker.ietf.org/doc/html/rfc7515):
+
+- `typ`: REQUIRED where the value must be `application/vnd.ga4gh.passport+jwt` for Passport tokens.
```

Comment:
The [RFC7515 section 4.1.9](https://datatracker.ietf.org/doc/html/rfc7515#section-4.1.9) recommends to omit the "application/" prefix of a media type value in a "typ" Header Parameter when no other '/' appears in the media type   value. So I think this should be just `vnd.ga4gh.passport+jwt`